packages/Material-Tools.package/MDLabel.class/instance/materialEmphasis.st

```diff
@@ -0,0 +1,4 @@
+scaling
+materialEmphasis
+
+	^ materialEmphasis ifNil: [materialEmphasis := #high]
```

Comment:
Maybe call the setter i/o direct field access?
```suggestion
	^ materialEmphasis ifNil: [self materialEmphasis := #high]
```

Reply:
This is normal Squeak style, since the assignment returns its value. Using the setter here would break the accessor, so we'll leave it as it is.